snapshot/git/gitfiler/setup.go

```diff
@@ -0,0 +1,52 @@
+package gitfiler
+
+import (
+	"io/ioutil"
+	"path"
+
+	"github.com/scootdev/scoot/os/temp"
+	"github.com/scootdev/scoot/snapshot/git/repo"
+)
+
+// Utilities for creating Checkouters.
+
+// RepoGetter lets a client get a Repository to use as a Reference Repository.
+type RepoGetter interface {
+	// Get gets the Repository to use as a reference repository.
+	Get() (*repo.Repository, error)
+}
+
+// A Pool that will only have a signle repo, populated by repoGetter, that serves as long as doneCh
```

Comment:
Err, I think I just wrote awkward English, because I thought I could omit the "as long as doneCh [is open]"